docs/variables.mk

```diff
@@ -1,11 +1,20 @@
+GIT_ROOT := $(shell git rev-parse --show-toplevel)
+
+MIMIR_DOCS_VERSION := $(shell sed -n 's, *MIMIR_DOCS_VERSION: "\([^"]*\)",\1,p' "$(GIT_ROOT)/docs/sources/helm-charts/mimir-distributed/_index.md")
+ifeq ($(MIMIR_DOCS_VERSION),next)
+MIMIR_DOCS_BRANCH := main
+else
+MIMIR_DOCS_BRANCH := $(patsubst v%.x,release-%,$(MIMIR_DOCS_VERSION))
+endif
+
 # List of projects to provide to the make-docs script.
-PROJECTS := mimir helm-charts/mimir-distributed
+PROJECTS := mimir:$(MIMIR_DOCS_VERSION):mimir-$(MIMIR_DOCS_VERSION) helm-charts/mimir-distributed
```

Comment:
This is missing "mimir", so you don't get the current mimir documentation on main rendered. Unfortunately if you do add mimir, it's rendered wrong. Latest mimir link will show the versioned documentation and there's errors like 
```
WARN  File "docs/mimir/latest/manage" not found, skipping.
WARN  File "docs/mimir/latest/query" not found, skipping.
WARN  File "docs/mimir/latest/send" not found, skipping.
WARN  File "docs/mimir/latest/set-up" not found, skipping.
WARN  File "docs/mimir/latest/visualize" not found, skipping.
```